w/wawa.md

```diff
@@ -0,0 +1,10 @@
+# Wawa
+
+![wawa](wawa.JPG)
+
+   This artist  is among the most popular contemporary salegy artists and has recorded songs in collaboration with numerous other Malagasy artists. In 2011 he released an album of traditional salegy, featuring kabosy, marovany and traditional percussion accompaniment.[2] In 2010 the band completed extensive tours to sold out audiences in France and Madagascar.[3] A music reviewer for Midi Madagasikara described Wawa in 2013 as the "perfect entertainer, who never ceases to perform at the highest levels.
```

Comment:
Please remove the spaces at the beginning of this paragraph.
Please also remove the `[1]` etc from the bio.

Reply:
I have removed the spaces, the description of the genre and the [1] etc from the bio.